client/src/components/Schedule/SettingsDrawer/SettingsDrawer.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Button, Drawer, Tooltip, Checkbox } from 'antd';
+import { SettingOutlined } from '@ant-design/icons';
+import { CheckboxChangeEvent } from 'antd/lib/checkbox';
+import { ScheduleSettings } from '../useScheduleSettings';
+import ChangeTagColors from './ChangeTagColors';
+import ShowTableColumns from './ShowTableColumns';
+import TaskLimits from './TaskLimits';
+
+interface SettingsDrawerProps {
+  settings: ScheduleSettings;
+  eventTypes: string[];
+}
+
+const TITLE = 'Schedule settings';
+
+const SettingsDrawer: React.FC<SettingsDrawerProps> = ({ settings, eventTypes }) => {
+  const [opened, setOpened] = useState(false);
+
+  const openDrawer = () => setOpened(true);
+  const closeDrawer = () => setOpened(false);
+  const toggleSplittedByWeek = ({ target }: CheckboxChangeEvent) => settings.setIsSplittedByWeek(target.checked);
+
+  return (
+    <Tooltip title={TITLE} mouseEnterDelay={1}>
+      <Button icon={<SettingOutlined />} title={TITLE} size="middle" type="primary" onClick={openDrawer} />
+      <Drawer title={TITLE} placement="right" closable={false} onClose={closeDrawer} visible={opened}>
+        <ShowTableColumns
+          eventTypes={eventTypes}
+          columnsHidden={settings.columnsHidden}
+          setColumnsHidden={settings.setColumnsHidden}
+          eventTypesHidden={settings.eventTypesHidden}
+          setEventTypesHidden={settings.setEventTypesHidden}
+        />
+        <ChangeTagColors tags={eventTypes} tagColors={settings.tagColors} setTagColors={settings.setTagColors} />
+        <TaskLimits limitForDoneTask={settings.limitForDoneTask} setLimitForDoneTask={settings.setLimitForDoneTask} />
+        <Checkbox value checked={settings.isSplittedByWeek} onChange={toggleSplittedByWeek}>
+          Splitted by week
+        </Checkbox>
+      </Drawer>
+    </Tooltip>
+  );
+};
```

Comment:
Consider avoid this duplication
![image](https://user-images.githubusercontent.com/34455330/162639753-d3173126-16f4-4427-a8b2-dededc40b0e6.png)

Reply:
By the way, is this expected behavior?
![image](https://user-images.githubusercontent.com/34455330/162639812-a2aea3e5-1db4-47f7-a840-5e04fed99514.png)

Reply:
It's working this way right now in the current version.

Reply:
Duplicated tooltip is deleted, thanks